packages/forms/resources/views/components/builder.blade.php

```diff
@@ -166,7 +166,7 @@
                                                     wire:target="dispatchFormEvent('builder::moveItemUp', '{{ $getStatePath() }}', '{{ $uuid }}')"
                                                     wire:loading.attr="disabled"
                                                     @class([
-                                                        'flex items-center justify-center flex-none w-10 h-10 text-gray-400 transition hover:text-gray-500',
+                                                        'flex items-center justify-center flex-none w-10 h-10 text-gray-400 transition outline-none hover:text-gray-500 focus:bg-gray-500/5 focus:ring-0',
```

Comment:
Have you checked dark mode?

Reply:
Tried to keep it simple with the semi-transparent background and reusing styling that are used elsewhere, but maybe it could be a lighter color for dark mode with brighter icon color?
<img width="868" alt="Screenshot 2023-05-03 at 16 17 18" src="https://user-images.githubusercontent.com/533658/235960762-5979a1cf-5578-447a-b2fa-30a862358def.png">